### PR TITLE
Seeds and api controlllers

### PR DESCRIPTION
**Pull request by kstensing**
seeds are now working simply run 'npm run seeds'

user routes are functional
get all users: 
image
get user by id: 
image
post (add) user: 
image

### CODE REVIEW

File: models/Post.js [hunk 1]
```diff
@@ -26,10 +26,9 @@ Post.init(
             type: DataTypes.STRING,
             allowNull: false
         },
-        private_post: {
-            type: DataTypes.BOOLEAN,
-            allowNull: false,
-            default: false
+        icon: {
+            type: DataTypes.STRING,
+            allowNull: false
```
Comment (nitrotap) [inline models/Post.js]: default: 'map-pin'

File: seeds/post-seeds.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  {
+    title: 'Guanella Pass',
+    contents: 'https://nasa.gov/donec.json',
+    location: '39.5950, 105.7111',
+    user_id: 8
+  },
+  {
+    title: 'Mammoth Cave',
+    contents: 'https://europa.eu/parturient/montes/nascetur/ridiculus/mus/etiam/vel.aspx',
+    location: ,
```
Comment (nitrotap) [inline seeds/post-seeds.js]: location: '37.179216, -86.1504',

adding location info for mammoth cave

File: seeds/post-seeds.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  {
+    title: 'Ashfall',
+    contents: 'The Ashfall Fossil Beds of Antelope County in northeastern Nebraska are rare fossil sites of the type called lagerstätten that, due to extraordinary local conditions, capture an ecological "snapshot" in time of a range of well-preserved fossilized organisms.',
+    location: '42.4217, 98.1563',
+    user_id: 10
+  },
+  {
+    title: 'Guanella Pass',
+    contents: 'https://nasa.gov/donec.json',
+    location: '39.5950, 105.7111',
```
Comment (nitrotap) [inline seeds/post-seeds.js]: location: '39.5950, -105.7111',

    location for W needs - symbol

File: seeds/post-seeds.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+const { Post } = require('../models');
+
+const postdata = [
+  {
+    title: 'Ashfall',
+    contents: 'The Ashfall Fossil Beds of Antelope County in northeastern Nebraska are rare fossil sites of the type called lagerstätten that, due to extraordinary local conditions, capture an ecological "snapshot" in time of a range of well-preserved fossilized organisms.',
+    location: '42.4217, 98.1563',
```
Comment (nitrotap) [inline seeds/post-seeds.js]: location: '42.4217, -98.1563',
    
    location for W needs - symbol

**Review by nitrotap — COMMENTED** [review]
redo to request changes

**Review by nitrotap — CHANGES_REQUESTED** [review]
if longitude includes W, it needs to have a (-) sign before it designating West

File: models/Post.js (hunk 1 above)
Comment (kstensing) [reply models/Post.js]: Got it - thanks! I'll add it

**Comment by kstensing** [comment]
> if longitude includes W, it needs to have a (-) sign before it designating West

Ohhhhhh I see what you were saying yesterday.  I've made the changes!

**Comment by kstensing** [comment]
Thanks for the review Kartik - I've made the changes please review

**Review by nitrotap — APPROVED** [review]
looks good!

**Pull request merged**